hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/client/ConfiguredRMFailoverProxyProvider.java

```diff
@@ -71,15 +71,9 @@ public void init(Configuration configuration, RMProxy<T> rmProxy,
             YarnConfiguration.DEFAULT_CLIENT_FAILOVER_RETRIES_ON_SOCKET_TIMEOUTS));
   }
 
-  protected T getProxyInternal() {
```

Comment:
if there is no actual change to this file, let's revert all the changes in this file.